src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10317,33 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
+    compiler->unwindPush(REG_FPBASE);
+
     // TODO Save callee-saved registers
 
     // This is the end of the OS-reported prolog for purposes of unwinding
     compiler->unwindEndProlog();
+
+    if (block->bbCatchTyp != BBCT_FINALLY && block->bbCatchTyp != BBCT_FAULT)
+    {
+        getEmitter()->emitIns_R_AR(INS_mov, EA_PTRSIZE, REG_EXCEPTION_OBJECT, REG_SPBASE, 12);
+
```

Comment:
@BruceForstall Thanks you for comment! It works well.
@wateret Please check the updated draft [here](https://github.com/parjong/coreclr/commit/b6e59da1c17ae4eb4f46549a8bca8c5a7e5164ca)